json-schema-store/src/main/java/uk/ac/ebi/biosamples/jsonschemastore/client/ValidatorClient.java

```diff
@@ -0,0 +1,31 @@
+package uk.ac.ebi.biosamples.jsonschemastore.client;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+import uk.ac.ebi.biosamples.jsonschemastore.client.dto.ValidateRequestDocument;
+import uk.ac.ebi.biosamples.jsonschemastore.client.dto.ValidateResponseDocument;
+
+import java.net.URI;
+
+@Slf4j
+public class ValidatorClient {
+
+  private static final String BASE_URL = "http://localhost"; // TODO: consume application properties
+  private static final String COLON = ":";
+  private static final String PORT = "3000";
```

Comment:
Nitpicking: :D we can have one variable to store complete base url with port

Reply:
review comment incorporated